webrtc.html

```diff
@@ -4450,6 +4450,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </li>
       </ul>
     </section>
+
+    <section>
+      <h3>RTCDataChannel and Workers</h3>
+      <p>An <code><a>RTCDataChannel</a></code> object MUST be
+      <code><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#transferable-objects">
+      Transferable</a></code> to <code>Workers</code> [[!WEBWORKERS]].  When it's
```

Comment:
`[Exposed]` doesn't need to change here.  We can expose `RTCDataChannel` in workers with no effort.

If the intent is to limit this to a worker (of what sort?!?), then we either have a mistake on the part of @jesup or maybe a misunderstanding about what transferable means.

My understanding is that a transferable object can be passed through `postMessage` to other origins as well as workers.  We should simply identify that `RTCDataChannel` as `[Transferable]` and note the consequences of that.